template/web/src/components/header/AccountInfoPanel.tsx

```diff
@@ -27,6 +32,14 @@ export function AccountInfoPanel() {
         </div>
       </div>
       <hr className="h-px self-stretch border-transparent bg-zinc-400 bg-opacity-20" />
+      <div className="inline-flex flex-col py-4">
+        <span className="font-inter w-32 text-sm font-medium text-zinc-400">{chain?.name}</span>
+        <span className="font-inter w-32 text-base font-medium text-white">
+          {formatEther(balanceResult.data?.value ?? 0n, 'wei').substring(0, 5)}{' '}
+          {balanceResult.data?.symbol}
```

Comment:
I believe this should be a part of onchain-kit in the avatar component where we can show the user's balance, what are your thoughts on this?

Reply:
yeah, makes sense to add an onchainkit equivalent here

Reply:
I believe we are building an opinionated codebase, so I believe it should be okay.

Reply:
PR in the onchainkit repo for this: https://github.com/coinbase/onchainkit/pull/155